jenkins/shared-library/vars/helper.groovy

```diff
@@ -109,3 +109,19 @@ boolean tryRunCommand(Closure command) {
 String resolveWorkspacePath(String os) {
 	return 'windows' == os ? 'C:\\Users\\Administrator\\jenkins\\workspace\\' : '/home/ubuntu/jenkins/workspace/'
 }
+
+boolean isGitHubRepositoryPublic(String orgName, String repoName) {
```

Comment:
this is already in githubHelper ?

Reply:
Yeah, this PR was before it got refactored into the gitHubHelper, which should be its final move.